dashboard/src/app/workspaces/workspace-details/workspace-machine-env-variables/edit-variable-dialog/edit-variable-dialog.html

```diff
@@ -26,12 +28,13 @@
       </che-label-container>
       <div layout="row" flex layout-align="end end">
         <che-button-primary che-button-title="{{editEnvVariableDialogController.toEdit ? 'Update' : 'Add'}}"
+                            id="{{editEnvVariableDialogController.toEdit ? 'Update' : 'Add'}}-dialog-button"
                             ng-disabled="envVariableForm.$invalid"
                             ng-click="editEnvVariableDialogController.saveVariable()"></che-button-primary>
 
         <che-button-notice che-button-title="Cancel"
                            ng-click="editEnvVariableDialogController.hide()"
-                           tabindex="0"></che-button-notice>
+                           tabindex="0"  id="Cancel-dialog-button"></che-button-notice>
```

Comment:
Could we name these ids at the same rule(Id name which starts from 'new-*' isn't correct - it can be an old one)?
We have  "new-variable-name"  and   "Cancel-dialog-button" in one file.